frontend/index.html

```diff
@@ -13,15 +15,7 @@
 
         <script src="bundle_min.js"></script>
         <!-- Google Analytics -->
-        <script>
-          (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-          (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-          m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-          })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-         ga('create', 'UA-64146053-1', 'auto');
-         ga('send', 'pageview');
-        </script>
+        <script src="{% static 'js/google-analytics.js' %}"></script>
```

Comment:
This doesn't work, because this file isn't a Django template.  The browser sees "{% load staticfiles %}" and "{% static '...' %} and fails to load it.

Hopefully this is the cause of the test errors on this branch?